common/viewer.h

```diff
@@ -118,6 +120,8 @@ namespace rs2
 
         void gc_streams();
 
+        bool skip_option(rs2_option opt);
```

Comment:
Returns whether the option is to be skipped in the GUI?
`is_option_skipped`?

Reply:
The function was there and was used, I just updated the content.
It used to support only common option skipped , I add the ability to skip only on DQT/VIEWER

I can rename it..

Reply:
If it's not too much work...

Reply:
👍